core/jazz_deployments-event-handler/test/test.js

```diff
@@ -622,8 +622,8 @@ describe("updateDeployments", () => {
     });
     index.updateDeployments(JSON.stringify(res), deploymentPayload, configData, "temp_auth").then((obj) => {
       sinon.assert.calledOnce(processRequestStub);
-      processRequestStub.restore();
     })
+    processRequestStub.restore();
```

Comment:
why was this moved out from the callback

Reply:
updated